sdk/tests/conformance/programs/00_test_list.txt

```diff
@@ -6,6 +6,7 @@ gl-get-active-uniform.html
 gl-getshadersource.html
 gl-shader-test.html
 invalid-UTF-16.html
+program-handling.html
```

Comment:
`--min-version 1.0.4` for new tests, sorry!

Reply:
Thanks for the instruction